scio-jdbc/src/test/scala/com/spotify/scio/jdbc/sharded/JdbcShardedSelectTest.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.jdbc
+package sharded
```

Comment:
is there a need for relative package?

Reply:
Usually I do this to avoid redundant import package prefixes e.g.:

```scala
package com.spotify.scio
package jdbc.sharded

import io.TextIO (instead of com.spotify.scio.io.TextIO)
````

but in this case it isn't needed, so I will remove this.

Reply:
In this case I bring everything in the scope of the `com.spotify.scio.jdbc` package so I don't need to import `CloudSqlOptions` and `JdbcJob`

Reply:
this is one of those things that is not really recommended for various reasons. one of them being bringing in an `io.example` dependency, suddenly you need to restructure this. Always prefer explicit import paths/package defs

Reply:
Agree, I had to change my habit of using this approach. It just so nice to omit imports if possible. Removed all relative package definitions.